contracts/factory/NexusAccountFactory.sol

```diff
@@ -42,14 +42,17 @@ contract NexusAccountFactory is Stakeable, INexusFactory {
     /// @param salt Unique salt for the Smart Account creation.
     /// @return The address of the newly created Nexus account.
     function createAccount(bytes calldata initData, bytes32 salt) external payable override returns (address payable) {
-        // Compute the actual salt for deterministic deployment
+        // Compute the actual salt as the hash of initData and salt using assembly
```

Comment:
whats wrong with this comment

// Compute the actual salt for deterministic deployment

Reply:
I changed it to reflect the changes, as we use hash of initData and Salt only (without sig func)

Reply:
@Aboudjem